app/components/pending-record/component.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
```

Comment:
Is there any reason all the `*-record` components couldn't be merged into a single one that accepts a `state` parameter?